src/components/accordion-card/accordion-card.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import chevronIcon from 'assets/icons/chevron.svg';
+import styles from './accordion-card-styles.scss';
+
+class AccordionCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isOpen: false };
+  }
+
+  componentDidMount() {
+    const { isOpen } = this.props;
+    this.setState({ isOpen });
+  }
+
+  render() {
+    const { icon, title, children, theme } = this.props;
+    const { isOpen } = this.state;
+    return (
+      <div
+        className={cx(styles.cardContainer, theme.cardContainer, {
+          [styles.open]: isOpen
+        })}
+      >
+        <div className={cx(styles.headerContainer, theme.headerContainer)}>
+          <h2 className={cx(styles.groupTitle, theme.groupTitle)}>
+            {title}
+          </h2>
+          <Button
+            onClick={() => this.setState({ isOpen: !isOpen })}
```

Comment:
Avoid the anonymous functions when possible in the render lifecycle, please!

Reply:
⚡️